CHANGELOG.md

```diff
@@ -71,7 +82,6 @@
 
 ### Feature
 
-- Added Italian translations and translated array, token and select widget. @giuliaghisini
```

Comment:
Seems there are some problems with the changeling. I can look into this before merging thg.